SecretSanta/src/SecretSanta.Domain/Models/Gift.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Domain.Models
+{
+    public class Gift : Entity
+    {
+        private string _Title;
+        public string Title
+        {
+            get => _Title;

```

Comment:
This is purely a style thing, but you can leave this line as auto implemented
```suggestion
            get;
```

Reply:
I tried this and Visual Studio yelled at me. Maybe because there's a private backing field the setter uses?